app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
-end
+  def x e
+    if !1 or 2
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Use || instead of or.